.travis.yml

```diff
@@ -1,6 +1,12 @@
 language: ruby
 sudo: required
 
+addons:
+  apt:
+    packages:
+      - unzip
+      - libaio-dev
```

Comment:
re `unzip`, that was my fault, sorry about that.

re `libaio1`, it is not in the [Travis apt whitelist](https://github.com/travis-ci/apt-package-whitelist), however `libaio-dev` is, and it pulls in `libaio1`.  We can request that `libaio1` be added to the whitelist...